install/ansible/install.sh

```diff
@@ -131,6 +131,9 @@ cp /var/contiv/key.pem /ansible/roles/auth_proxy/files/
 if [ "$aci_image" != "" ]; then
 	sed -i.bak "s#.*aci_gw_image.*#\"aci_gw_image\":\"$aci_image\",#g" "$env_file"
 fi
+if [ "$contiv_v2plugin_install" == "true" ]; then
```

Comment:
Should it not be exclusive with -i option? AFAIK, there isn't a point in installer covering the docker swarm install and v2plugin install option (even though they can be used together).